lib/mvi/messages/find_profile_message_helpers.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'ox'
+require_relative 'message_builder'
+require_relative 'find_candidate_message_error'
+
+module MVI
+  module Messages
+    module FindProfileMessageHelpers
+      include MVI::Messages::MessageBuilder
+      EXTENSION = 'PRPA_IN201305UV02'
```

Comment:
This is still a 1305 request to MVI, so while I will still test it in prod via console, I'm less concerned. I thought there was a use case where we needed to use the 1309 GetCorrespondingIds method, but if we're always either searching by traits or ICN, then I guess no.t